preprocess.py

```diff
@@ -0,0 +1,17 @@
+
+def read_text(filename: str) -> str:
+    """
+        This function reads a file given a file path and converts contents into a string. 
+    """
+    with  open(filename,'r',encoding='utf-8',newline='\n') as file:
+        
```

Comment:
for our project, we should decide on how we should do spacing so that we are all consistent :)

Reply:
Yeah that would a good idea! Maybe we format that once we get more stuff pushed and merged into master